e2e-tests/Jenkinsfile

```diff
@@ -15,12 +15,8 @@ pipeline {
     timeout(time: 1, unit: 'HOURS')
   }
   environment {
-    PYTEST_ADDOPTS =
-      "-n=10 " +
-      "--color=yes " +
-      "--tb=short " +
-      "--driver=SauceLabs " +
-      "--variables=capabilities.json"
+    /** See https://issues.jenkins-ci.org/browse/JENKINS-42857 - we'd like to expand this out into multi-line concatenations */
+    PYTEST_ADDOPTS = "-n=10 --tb=short --color=yes --driver=SauceLabs --variables=capabilities.json"
```

Comment:
This line isn't really that long, but I definitely find it easier to read the command line arguments when they're split over multiple lines. If others are fine with a single line then we can even remove the comment and just go with that.